docs/built-in-functions.rst

```diff
@@ -622,4 +622,28 @@ Utilities
     .. code-block:: python
 
         >>> ExampleContract.foo()
-        b"\xa9\x05\x9c\xbb"
+
+.. py:function:: _abi_encode(\*args, ensure_tuple: bool = True) -> Bytes[<depends on input>]
+
+    BETA, USE WITH CARE.
+    Takes a variable number of args as input, and returns the ABIv2-encoded bytestring. Used for packing arguments to raw_call, EIP712 and other cases where a consistent and efficient serialization method is needed.
+    Once this function has seen more use we provisionally plan to put it into the ethereum.abi namespace.
+
+    * ``*args``: Arbitrary arguments
+    * ``ensure_tuple``: If set to True, ensures that even a single argument is encoded as a tuple. In other words, `bytes` gets encoded as `(bytes)`. This is the calling convention for Vyper and Solidity functions. Except for very specific use cases, this should be set to True. Must be a literal.
+
+    Returns a bytestring whose max length is determined by the arguments. For example, encoding a Bytes[32] results in a Bytes[64] (first word is the length of the bytestring).
+
+    .. code-block:: python
+
+        @external
+        @view
+        def foo() -> Bytes[4]:
+            x: uint256 = 1
+            y: Bytes[32] = "234"
+            return _abi_encode(x, y)
```

Comment:
is this correct? Shouldn't it return `Bytes[96]`?

Reply:
Weird, I must have meant 4 words.

128 bytes. word by word, that's
```
x | <position of y in dynamic data> | len(y) | data(y)
```